vignettes/blueprint/in_app_data.Rmd

```diff
@@ -0,0 +1,26 @@
+---
+title: "In-App Data"
+author: "NEST CoreDev"
+output: rmarkdown::html_vignette
```

Comment:
would you mind adding `description` field for the new / significantly modified vignettes? I will create a separate issue to go through all the vignettes (of all packages).

This will render nicely in the `(..,)/articles/index.html` webpage to which you put quite a few references. So if other pkgs do the same then those vignettes will have short summary making it easier to navigate.

Example: https://dplyr.tidyverse.org/articles/index.html

Reply:
I don't really like it. It creates a redundant navigation route (the `Articles` dropdown still exists) and it adds redundant reading. I think our titles are (mostly) good enough. Also, wouldn't that replace the highly elaborate README?

Also it's out of scope.

I suggest creating a separate issue to encompass vignettes in _all_ packages.